timeseries/src/autogluon/timeseries/models/ensemble/greedy_ensemble.py

```diff
@@ -110,27 +110,14 @@ def predict(self, data: Dict[str, TimeSeriesDataFrame], **kwargs) -> TimeSeriesD
             raise ValueError(
                 f"Set of models given for prediction in {self.name} differ from those provided during initialization."
             )
-        failed_models = [model_name for (model_name, model_pred) in data.items() if model_pred is None]
-        if len(failed_models) == len(data):
-            raise RuntimeError(f"All input models failed during prediction, {self.name} cannot predict.")
-        if len(failed_models) > 0:
-            logger.warning(
-                f"Following models failed during prediction: {failed_models}. "
-                f"{self.name} will set the weight of these models to zero and re-normalize the weights when predicting."
-            )
+        for model_name, model_pred in data.items():
+            if model_pred is None:
+                raise RuntimeError(f"{self.name} cannot predict because base model {model_name} failed.")
```

Comment:
nit: Consider listing all failed model names instead of only the first failed model. This will help to identify bugs during development easier.